tests/xtimer_usleep_short/main.c

```diff
@@ -22,18 +22,54 @@
 #define TEST_USLEEP_MIN (0)
 #define TEST_USLEEP_MAX (500)
 
+#ifdef BOARD_NATIVE
+/* native can sometime take more time to respond as it is not real time */
+#define TEST_XTIMER_USLEEP_SHORT_SLEEP_MARGIN_US (1000)
+#else
+#define TEST_XTIMER_USLEEP_SHORT_SLEEP_MARGIN_US (20)
```

Comment:
The margin may not be enough as mentioned in #8990.
I need to do the xtimer_configure to check if it helps getting in the margin.